### PR TITLE
Create a Recipient page to create recipient offer

### PR DESCRIPTION
**Pull request by Eugene-pas**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ import ConfirmEmailPage from "./components/emailConfirmation";
 import ManageUsersPage from './components/users/index';
 import UserCarsPage from "./components/userCars";
 import CreateOfferPage from "./components/offerPage";
+import {offerRoles} from "./constants/offerRoles";
```
Comment (Grasmus) [inline src/App.js]: Remove empty line

File: src/services/offers.js [hunk 2]
```diff
@@ -33,13 +32,15 @@ export async function getUserOffers(paginationFilterModel) {
 }
 
 export function createOffer(values, coordinates, history) {
+    console.log(values)
```
Comment (Grasmus) [inline src/services/offers.js]: Remove this

File: src/components/offerPage/index.jsx [hunk 3]
```diff
@@ -173,14 +173,14 @@ export default function CreateOfferPage() {
         const start = values.dates[0];
         const end = values.dates[1];
         const diff = end.diff(start, 'hours');
-
+        const createValue = {...values, role: props.offerRole};
```
Comment (Grasmus) [inline src/components/offerPage/index.jsx]: Why did you name variable as "createValue"? What does this variable do?

**Review by Grasmus — APPROVED** [review]

File: src/constants/offerRoles.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
 export const offerRoles = {
-    SENDER: "SENDER",
-    RECIPIENT: "RECIPIENT"
+    SENDER: "sender",
+    RECIPIENT: "recipient"
```
Comment (marynakernychna) [inline src/constants/offerRoles.js]: Return as it was

**Review by marynakernychna — APPROVED** [review]

**Pull request merged**